Integrations/integration-UrlScan.yml

```diff
@@ -586,17 +585,14 @@ script:
       defaultValue: "20"
     outputs:
     - contextPath: URL.Data
-      description: Bad URLs found
+      description: URL submitted
       type: string
     - contextPath: URL.Malicious.Vendor
       description: For malicious URLs, the vendor that made the decision
       type: string
     - contextPath: URL.Malicious.Description
       description: For malicious URLs, the reason for the vendor to make the decision
       type: string
-    - contextPath: URLScan.URLs
```

Comment:
shouldn't we have this? there is a setting for cont['URL'], which prob mean we should get URLScan.URL, no?